DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Controllers/RepositoryController.cs

```diff
@@ -207,5 +210,28 @@ private void AddClaimsToCookie(Claim[] claims)
                 GetRequestContext.SetCookie(claim.Type, claim.Value);
             }
         }
+
+        IEnumerable<CollaboratorWithUserIdModel> AddUserIdToCollaboratorIfExists(
```

Comment:
I don't undertand why it can be moved to extension method? Why you need this logic inside controller?